### DISCUSSION TITLE
yScale/xScale on LinePath

### DISCUSSION THREAD

**Discussion by kaisiant** [airbnb/visx#1022] [Q&A] [answered]
Hello, I'm new to this library and D3. I went through a demo https://github.com/hshoff/viewsource#1-bitcoin-price-chart, seems the LinePath component no longer accepts y/xScale. Is there any latest example for me as a reference? thanks

![image](https://user-images.githubusercontent.com/26655111/104532436-319f6e00-564b-11eb-8649-5bcb499eea21.png)

**Comment by williaster** [comment, marked as answer]
Hey @kaisian97 thanks for checking out `visx`! I think that example was created before the project was re-written in TypeScript and we tightened up some APIs.

In any case, if you're on the latest `@visx/*` versions, you can check out this [`/curves`](https://airbnb.io/visx/curves) example from the gallery (and the sandbox link on that page if you want to tinker). **tl;dr** currently only `x/y` are valid props, and so you would just use the scale at that point:

```tsx
<LinePath<Datum>
  curve={...}
  data={lineData}
  x={(d: Datum) => xScale(getX(d)) ?? 0}
  y={(d: Datum) => yScale(getY(d)) ?? 0}
  {...
/>
```
